week_4/project/week_4.py

```diff
@@ -43,6 +46,11 @@ def put_redis_data(context, process_data):
     definitions=[get_s3_data, process_data, put_redis_data],
     resource_defs={"redis": redis_resource, "s3": s3_resource},
     resource_config_by_key={
+        # "key_name": {
+        #     "config": {
+        #         "s3_key": "prefix/stock.csv"
+        #     }
+        # },
```

Comment:
@dfg129 you can study the test and hopefully it will help you to fix the recource_config error - https://github.com/dehume/corise-dagster/blob/750ee612d6a525bffd0ecb42cd650996cb1d85d3/week_4/tests/test_answer.py#L37. You are pretty close.